scripts/generate-react-exports.js

```diff
@@ -25,6 +25,7 @@ const baklavaReactFileParts = [];
 for (const module of customElementsModules) {
   const { declarations, path } = module;
   const component = declarations.find(declaration => declaration.customElement === true);
+  if(!component) continue;
```

Comment:
when this case may be possible?

Reply:
build gives error for calendar's types and constants files so we add this with @Enes5519

Reply:
Ok, I understand your concern but I have better idea. `custom-elements-manifest/analyzer` package generates `custom-elements.json` file. You got error because there are modules that created in custom-elements.json file about `bl-calendar.constant.ts` and `bl-calendar.types.ts`. 
When we run `build` command (..`npm run analyze && npm run generate-react-exports`..) first it generates custom-elements.json and generate-react-export script uses that file. So we have `custom-elements-manifest.config.mjs` file to exclude unnecessary files like css, types, constants etc.

```
export default {
  globs: ["src/components/**/!(*.test|*.stories).ts"],
  exclude: ["src/**/*.css","src/components/icon/icon-list.ts"], // we can exclude unnecessary files for custom-elements.json file. We already exclude css files for example.
....
```
We should add pattern for types and constants files. So, final version should be like below:
```
export default {
  globs: ["src/components/**/!(*.test|*.stories).ts"],
  exclude: ["src/**/*.css", "src/**/*.constant.ts","src/**/*/*.types.ts","src/components/icon/icon-list.ts"],
....
```